components/WorkoutForm/index.js

```diff
@@ -105,42 +106,43 @@ export default function WorkoutForm({
             placeholder="Insert name here"
             defaultValue={defaultData?.name}
             required
+            $edit={isEditMode}
           />
-        </StyledSection>
-        <h3>Add your exercises</h3>
-        <StyledSection>
-          <h4>Added exercises:</h4>
-          <ExercisesList>
-            {addedExercises.map((addedExercise) => (
-              <li key={addedExercise.id}>
-                {addedExercise.exercise} sets: {addedExercise.sets} reps:{" "}
-                {addedExercise.reps}
-                <DeleteButton
-                  type="button"
-                  onClick={() => handleDeleteExercise(addedExercise.id)}
-                >
-                  X
-                </DeleteButton>
-              </li>
-            ))}
-          </ExercisesList>
-          <label htmlFor="exerciseName">Exercise name:</label>
+        </StyledFieldset>
+        <StyledFieldset $edit={isEditMode}>
+          <StyledLegend $edit={isEditMode}>Second step</StyledLegend>
+          <ExercisesListHeadline>Added to list:</ExercisesListHeadline>
+          {addedExercises.map((addedExercise) => (
+            <AddedExercise key={addedExercise.id}>
+              {addedExercise.exercise} {addedExercise.sets} sets /{" "}
+              {addedExercise.reps} reps
+              <DeleteButton
+                type="button"
+                onClick={() => handleDeleteExercise(addedExercise.id)}
+                $edit={isEditMode}
+              >
+                X
+              </DeleteButton>
+            </AddedExercise>
+          ))}
+          <StyledLabel htmlFor="exerciseName">Exercise name</StyledLabel>
           <StyledDropdown
             name="exerciseName"
             id="exerciseName"
             required={addedExercises.length === 0 ? true : false}
```

Comment:
If you have comparisons like this, you can write them shorter, since a comparison always evaluates to `true` or `false`:

```suggestion
            required={addedExercises.length === 0}
```